cli-plugins/apps/how-to-guides.hbs.md

```diff
@@ -0,0 +1,611 @@
+                        "resource": "gitrepositories.source.toolkit.fluxcd.io"
+                    },
+                    "templateRef": {
+                        "kind": "ClusterSourceTemplate",
+                        "name": "source-template",
+                        "apiVersion": "carto.run/v1alpha1"
+                    },
+                ...
+                }
+            ...
+            ]
+            ...
+        }
+        ...
+    }
+
+## with yaml format
+tanzu apps workload get rmq-sample-app --output yaml # can also be used as tanzu apps workload get rmq-sample-app -oyaml
+---
+apiVersion: carto.run/v1alpha1
+kind: Workload
+metadata:
+  creationTimestamp: "2022-11-28T05:10:32Z"
+  generation: 2
+  labels:
+    apps.tanzu.vmware.com/workload-type: web
+  managedFields:
+  - apiVersion: carto.run/v1alpha1
+      ...
+    manager: v0.10.0+dev-002cc44e
+    operation: Update
+    time: "2022-11-28T05:10:32Z"
+  - apiVersion: carto.run/v1alpha1
+    fieldsType: FieldsV1
+    ...
+    manager: cartographer
+    operation: Update
+    subresource: status
+    time: "2022-11-28T05:10:36Z"
+  name: rmq-sample-app
+  namespace: default
+  resourceVersion: "11657434"
+  uid: 3619ff6d-9e73-473a-9112-891a6d8aee9e
+spec:
+  serviceClaims:
+  - name: rmq
+    ref:
+      apiVersion: rabbitmq.com/v1beta1
+      kind: RabbitmqCluster
+      name: example-rabbitmq-cluster-1
+  source:
+    git:
+      ref:
+        branch: main
+      url: https://github.com/jhvhs/rabbitmq-sample
+status:
+  conditions:
+  - lastTransitionTime: "2022-11-28T05:10:32Z"
+    message: ""
+    reason: Ready
+    status: "True"
+    type: SupplyChainReady
+  ...
+  observedGeneration: 2
+  resources:
+  ...
+    name: source-provider
+    outputs:
+    - digest: sha256:97b2cb779b4ea31339595cd204a3fec0053805eeacbbd6d6dd23af7d3000a6ae
+      lastTransitionTime: "2022-11-28T05:16:01Z"
+      name: url
+      preview: |
+        http://fluxcd-source-controller.flux-system.svc.cluster.local./gitrepository/default/rmq-sample-app/73c6311eefbf724fee9ad6f4524fa24ec842ff34.tar.gz
+    - digest: sha256:e7884b071fe1bbb2551d42a171043d061a7591e744705572136e689c2a154b7a
+      lastTransitionTime: "2022-11-28T05:16:01Z"
+      name: revision
+      preview: |
+        HEAD/73c6311eefbf724fee9ad6f4524fa24ec842ff34
+    stampedRef:
+      apiVersion: source.toolkit.fluxcd.io/v1beta1
+      kind: GitRepository
+      name: rmq-sample-app
+      namespace: default
+      resource: gitrepositories.source.toolkit.fluxcd.io
+    templateRef:
+      apiVersion: carto.run/v1alpha1
+      kind: ClusterSourceTemplate
+      name: source-template
+  - conditions:
+    - lastTransitionTime: "2022-11-28T05:13:25Z"
+      message: ""
+      reason: ResourceSubmissionComplete
+      status: "True"
+      type: ResourceSubmitted
+    ...
+    inputs:
+    - name: source-provider
+```
+
+## <a id='subpath-usage'> --sub-path
+
```

Comment:
This flag is provided to support use cases where more than one application is included in a single project or repository.